radixdlt-core/radixdlt/src/main/java/com/radixdlt/store/LedgerEntryStoreView.java

```diff
@@ -45,6 +45,15 @@
 	 */
 	Optional<Atom> get(AID aid);
 
+	/**
+	 * Gets the atom directly from commit log.
+	 *
+	 * @param address
+	 * 			Address to read atom from.
+	 * @return the atom stored at given address
+	 */
+	Optional<Atom> get(long address);
```

Comment:
This isn't used. Please delete.

Reply:
Yep, it's a leftover from previous design. Fixed.